src/west/config.py

```diff
@@ -0,0 +1,56 @@
+'''
+Configuration file handling, using the standard configparser module.
+'''
+
+import configparser
+import os
+import platform
+
+from west.util import west_dir
+
+
+def read_config():
+    '''
+    Reads all configuration files, making the configuration values available as
+    a configparser.ConfigParser object in config.config. This object works
+    similarly to a dictionary: config.config['foo']['bar'] gets the value for
+    key 'bar' in section 'foo'.
+    '''
+
+    global config
+
+    #
+    # Gather (potential) configuration file paths, roughly following Git
```

Comment:
The reason that I specifically chose to put the configuration file locations in a module level docstring in issue https://github.com/zephyrproject-rtos/west/issues/67 is because we are going to have to document this behavior for users, and it opens the path to letting a sphinx pydoc directive help us do that in a clean way. Doing it in plain comments will mean we have to duplicate the information in .rst, which encourages docs getting stale.

Reply:
Agree with that, let's mirror your text from #67 with the new locations in this PR

Reply:
Ah, didn't notice those little `'''` around it.

Yeah, nice to have it in the docstring. I'll add it.

Reply:
Done

Reply:
> Ah, didn't notice those little ''' around it.

I guess you also missed the text "Here is a proposed top-level docstring" which appeared before it.

Thanks for making this change.